### ISSUE TITLE
Python syntax highlight

### ISSUE THREAD

**Issue by asm0dey** [hiroppy/fusuma#223]
## Info

- Operating System: Manjaro Linux
- Node Version: v13.5.0
- Browser Name and Version: Firefox 71.0

## Type

- [x] bug
- [ ] feature request

```jsx
<Code language="python">{`def get_book(*, id: uuid.UUID) -> Book:
    book = Book.objects.get(id=id)
    author = AuthorInterface.get_author(id=book.author_id)
    return {
        'name': book.name,
        'author_name': author.name,
    }
`}</Code>
```

### Expected Behavior

Should highlight python code

### Actual Behavior

doesn't highlight python code

**Comment by hiroppy** [comment]
See https://hiroppy.github.io/fusuma/docs/guides/configuring-fusuma#code-syntax-highlighting

**Comment by asm0dey** [comment]
Sorry, I don't understand. I've added python support into .fusumarc. What else should I do to get python syntax highlighting?

**Comment by daviddahl** [comment]
Yes, this does not work by default, after an hour of reading and configuring, it is not clear how to make python code highlighting work.

i was just trying:

```
```python
self.foo = 'bar'
```

```

**Comment by asm0dey** [comment]
@hiroppy please reopen, it doesn't work.

**Comment by hiroppy** [comment]
Ok, I opened and I'm going to investigate this issue.

**Comment by hiroppy** [comment]
I fixed this issue. see https://deploy-preview-234--heuristic-davinci-1cf783.netlify.com/#slide=8

**Comment by hiroppy** [comment]
I've just released v1.15.2. Thank for everyone!
